### PR TITLE
get rid of asctec_hl_comm dependency

### PR DESCRIPTION
**Pull request by ffurrer**
@markusachtelik can you take a look at this please

### CODE REVIEW

File: msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp [hunk 1]
```diff
@@ -30,8 +30,8 @@ PressureSensorHandler::PressureSensorHandler(
   ros::NodeHandle pnh("~/pressure_sensor");
   ros::NodeHandle nh("msf_updates");
   subPressure_ =
-      nh.subscribe<asctec_hl_comm::mav_imu>
-      ("pressure_input", 20, &PressureSensorHandler::MeasurementCallback, this);
+      nh.subscribe<geometry_msgs::PointStamped>
+      ("pressure_height", 20, &PressureSensorHandler::MeasurementCallback, this);
```
Comment (markusachtelik) [inline msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp]: make sure you change the topic name everywhere

**Comment by markusachtelik** [comment]
thanks, lgtm :)

**Comment by goldbattle** [comment]
Good stuff :+1:

File: msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp (hunk 1 above)
Comment (ffurrer) [inline msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp]: good point, I think [here](https://github.com/ethz-asl/ethzasl_msf/blob/master/msf_updates/launch/pose_pressure_sensor.launch#L6) is another occurrence, but I would prefer if we could quickly try if everything works on an actual platform, before merging this.

Comment (markusachtelik) [inline msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp]: I'd say change it in the launchfile, then we can merge this. We still need to implement it in asctec_mav_framework

**Comment by ffurrer** [comment]
test this please

**Comment by simonlynen** [comment]
@ffurrer @markusachtelik thanks

**Pull request merged**